setup.py

```diff
@@ -54,7 +50,7 @@
 
 setup(
     name='google-gax',
-    version=version,
+    version='0.15.5',
```

Comment:
No, I was just going to let it sit until the next thing happens (mostly because "the next thing" I need to do in here will probably be within the next 2-3 days).